<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuWindow</class>
 <widget class="QMainWindow" name="StuWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="UserInfo">
     <item>
      <widget class="QLabel" name="acclabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="namelabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上传实验报告</string>
    </property>
   </widget>
   <widget class="QPushButton" name="record">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="RecordStatus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RecordStatus</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ExpSelectComboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1、碳酸纳与盐酸的反应</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2、反应热测定</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3、碳酸氢钠与不同浓度盐酸反应</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4、绘制中和滴定曲线</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
